src/Sulu/Bundle/SearchBundle/Tests/Functional/Controller/SearchControllerTest.php

```diff
@@ -41,6 +41,17 @@ class SearchControllerTest extends SuluTestCase
      */
     private $user;
 
+    public function setUp()
+    {
+        parent::setUp();
+        $this->purgeDatabase();
+        $this->client = $this->createAuthenticatedClient();
+        $this->searchManager = $this->client->getContainer()->get('massive_search.search_manager');
```

Comment:
Would rather not engage in writing a chapter on how to write tests for this PR .. will expand the php doc instead.